{
  "uid" : "6dd64adc1f1e2f4",
  "name" : "Unsuccessfully validating product by ID in the URL",
  "fullName" : "Integration-Tests.Produtos.TestProdutos.Unsuccessfully validating product by ID in the URL",
  "historyId" : "ec9c8b0dd38791113417afb390c60b2b",
  "time" : {
    "start" : 1655220759251,
    "stop" : 1655220759867,
    "duration" : 616
  },
  "status" : "failed",
  "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f2538fa00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
  "flaky" : true,
  "newFailed" : true,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f2538fa00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
    "steps" : [ {
      "name" : "ProdutoKW.Search Products By Id In The URL",
      "time" : {
        "start" : 1655220759252,
        "stop" : 1655220759865,
        "duration" : 613
      },
      "status" : "failed",
      "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f2538fa00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "steps" : [ {
        "name" : "BuiltIn.Create Dictionary",
        "time" : {
          "start" : 1655220759252,
          "stop" : 1655220759253,
          "duration" : 1
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ {
          "uid" : "df578d04bbf04c52",
          "name" : "Keyword Log",
          "source" : "df578d04bbf04c52.html",
          "type" : "text/html",
          "size" : 75
        } ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "content-type=application/json"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      }, {
        "name" : "RequestsLibrary.Create Session",
        "time" : {
          "start" : 1655220759253,
          "stop" : 1655220759254,
          "duration" : 1
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ {
          "uid" : "c33720dc0669566b",
          "name" : "Keyword Log",
          "source" : "c33720dc0669566b.html",
          "type" : "text/html",
          "size" : 219
        } ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "serveRest"
        }, {
          "name" : "arg2",
          "value" : "${BASE_URL}"
        }, {
          "name" : "arg3",
          "value" : "verify=True"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      }, {
        "name" : "RequestsLibrary.GET On Session",
        "time" : {
          "start" : 1655220759254,
          "stop" : 1655220759863,
          "duration" : 609
        },
        "status" : "failed",
        "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f2538fa00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "steps" : [ ],
        "attachments" : [ {
          "uid" : "65f110fc7805dd37",
          "name" : "Keyword Log",
          "source" : "65f110fc7805dd37.html",
          "type" : "text/html",
          "size" : 1296
        } ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "serveRest"
        }, {
          "name" : "arg2",
          "value" : "/produtos/${product_id}"
        }, {
          "name" : "arg3",
          "value" : "headers=${headers}"
        }, {
          "name" : "arg4",
          "value" : "expected_status=any"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      }, {
        "name" : "BuiltIn.Log",
        "time" : {
          "start" : 1655220759863,
          "stop" : 1655220759864,
          "duration" : 1
        },
        "status" : "failed",
        "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f2538fa00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "${response}"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      }, {
        "name" : "BuiltIn.Set Test Variable",
        "time" : {
          "start" : 1655220759864,
          "stop" : 1655220759864,
          "duration" : 0
        },
        "status" : "failed",
        "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f2538fa00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "${response}"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "arg1",
        "value" : "BeeJh5lz3k6kSIzB"
      } ],
      "stepsCount" : 5,
      "attachmentsCount" : 3,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "BuiltIn.Should Be Equal As Strings",
      "time" : {
        "start" : 1655220759865,
        "stop" : 1655220759866,
        "duration" : 1
      },
      "status" : "failed",
      "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f2538fa00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "arg1",
        "value" : "${response.status_code}"
      }, {
        "name" : "arg2",
        "value" : "400"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "BuiltIn.Should Be Equal As Strings",
      "time" : {
        "start" : 1655220759866,
        "stop" : 1655220759866,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1f2538fa00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "arg1",
        "value" : "${response.json()[\"message\"]}"
      }, {
        "name" : "arg2",
        "value" : "Produto n√£o encontrado"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 8,
    "attachmentsCount" : 3,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "Integration-Tests"
  }, {
    "name" : "suite",
    "value" : "Produtos"
  }, {
    "name" : "subSuite",
    "value" : "TestProdutos"
  }, {
    "name" : "framework",
    "value" : "robotframework"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "host",
    "value" : "fv-az180-72"
  }, {
    "name" : "thread",
    "value" : "default-bcd441f3-51a0-4588-8b56-57009bb64496"
  }, {
    "name" : "tag",
    "value" : "@integration_tests"
  }, {
    "name" : "tag",
    "value" : "@produtos"
  }, {
    "name" : "tag",
    "value" : "@regression"
  }, {
    "name" : "tag",
    "value" : "@integration_tests"
  }, {
    "name" : "tag",
    "value" : "@produtos"
  }, {
    "name" : "tag",
    "value" : "@regression"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 4,
        "unknown" : 0,
        "total" : 7
      },
      "items" : [ {
        "uid" : "8e060b97e649ec9c",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/9//#testresult/8e060b97e649ec9c",
        "status" : "passed",
        "time" : {
          "start" : 1655219662847,
          "stop" : 1655219662941,
          "duration" : 94
        }
      }, {
        "uid" : "40ac3f1e44527585",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/8//#testresult/40ac3f1e44527585",
        "status" : "passed",
        "time" : {
          "start" : 1655218423529,
          "stop" : 1655218423605,
          "duration" : 76
        }
      }, {
        "uid" : "6af615396119078a",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/7//#testresult/6af615396119078a",
        "status" : "passed",
        "time" : {
          "start" : 1655217744733,
          "stop" : 1655217744826,
          "duration" : 93
        }
      }, {
        "uid" : "76d7a4c38fc2de",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/5//#testresult/76d7a4c38fc2de",
        "status" : "failed",
        "statusDetails" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')))",
        "time" : {
          "start" : 1655217412497,
          "stop" : 1655217413114,
          "duration" : 617
        }
      }, {
        "uid" : "500f9a14c963a200",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/4//#testresult/500f9a14c963a200",
        "status" : "failed",
        "statusDetails" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos/BeeJh5lz3k6kSIzB (Caused by ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')))",
        "time" : {
          "start" : 1655217118026,
          "stop" : 1655217118642,
          "duration" : 616
        }
      }, {
        "uid" : "f6e29ad1a7f89e54",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/3//#testresult/f6e29ad1a7f89e54",
        "status" : "passed",
        "time" : {
          "start" : 1655216713300,
          "stop" : 1655216713315,
          "duration" : 15
        }
      } ]
    },
    "tags" : [ "@regression", "@integration_tests", "@produtos" ]
  },
  "source" : "6dd64adc1f1e2f4.json",
  "parameterValues" : [ ]
}